casa/System/AppState.h

```diff
@@ -65,6 +65,14 @@ class AppState {
         return result;
     }
 
+    // get directory containing IERS measures data
+    // an exception is thrown if it does not exist or
+    // does not contain the IERS tables
```

Comment:
Suggested comment: get AppState specified directory for (IERS) measures data. If data is not found in the specified directory and the specified directiory name is nonempty, an exception will be thrown in findTab.

Reply:
Thanks! Sounds good to me. I'll push up the change.